Tests/Fixtures/Generated/XCAssets/swift4-context-all-forceNamespaces.swift

```diff
@@ -45,14 +42,14 @@ internal struct DataAsset {
 
   #if os(iOS) || os(tvOS) || os(OSX)
   @available(iOS 9.0, tvOS 9.0, OSX 10.11, *)
-  internal var data: AssetDataTypeAlias {
-    return AssetDataTypeAlias(asset: self)
+  internal var data: NSDataAsset {
+    return NSDataAsset(asset: self)
   }
   #endif
 }
 
 #if os(iOS) || os(tvOS) || os(OSX)
-internal extension AssetDataTypeAlias {
+internal extension NSDataAsset {
```

Comment:
Now that I think about it, shouldn't this also be surrounded by platform checks? So, move the `@available` up to mark this extension, instead of the `init`.

Reply:
The documentation about this, and SO answers, are a bit confusing...

Reply:
Swift compiler OTOH is pretty clear (tested in a small project): the `@available` must be on the extension.

Reply:
That's unfortunate that none of our tests compiling the generated output caught that mistake. Any idea why? Maybe because it's available in macOS and iOS just not on watchOS which might be the only one we don't test?

Reply:
We test all platforms (even watchOS and tvOS), but I think we have a "default" deployment target of "latest iOS/macOS/..."